package.json

```diff
@@ -67,6 +68,7 @@
     "mocha": "3.4.2",
     "ngrok": "2.2.10",
     "node-watch": "0.5.4",
+    "require-from-string": "2.0.1",
```

Comment:
This package is used to load the rendered auth code, i.e., `authentication.js`. Since `authentication.js` is a JavaScript module (with `module.exports = ...` at the end), we can't just use `s2js()` or `eval()` to evaluate the code. `require-from-string` allows us to do `require()` but from a string instead of a file.